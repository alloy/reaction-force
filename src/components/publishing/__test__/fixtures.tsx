export const Images = [
  {
    type: "artwork",
    id: "589a6291275b2410d1beb6a5",
    slug: "fernando-botero-nude-on-the-beach",
    date: "2000",
    title: "Nude on the Beach",
    image: "https://d32dm0rphc51dk.cloudfront.net/0aRUvnVgQKbQk5dj8xcCAg/larger.jpg",
    partner: {
      name: "Gary Nader",
      slug: "gary-nader",
    },
    artists: [
      {
        name: "Fernando Botero",
        slug: "fernando-botero",
      },
    ],
    artist: {
      name: "Fernando Botero",
      slug: "fernando-botero",
    },
    width: 1152,
    height: 826,
  },
  {
    url: "https://artsy-media-uploads.s3.amazonaws.com/co8j2xq40ROMyBrJQm_4eQ%2FDafenOilPaintingVillage_AK03.jpg",
    type: "image",
    width: 900,
    height: 1200,
    caption: "<p>Photo by Adam Kuehl for Artsy.</p>",
  },
  {
    url: "https://d32dm0rphc51dk.cloudfront.net/CpHY-DRr7KW0HGXLslCXHw/larger.jpg",
    type: "image",
    width: 816,
    height: 1024,
    caption: "<p>Photo by Adam Kuehl for Artsy. Image courtesy of the Guggenheim Museum.</p>",
  },
]

export const ImageSetFull = {
  type: "image_set",
  layout: "full",
  title: "The Work of Bruce M. Sherman",
  images: Images,
}

export const ImageSetMini = {
  type: "image_set",
  layout: "mini",
  title: "The Work of Bruce M. Sherman",
  images: Images,
}

export const FeatureHeaders = [
  {
    layout: "text",
    title: "Looking Into the Future",
    url: "https://artsy-media-uploads.s3.amazonaws.com/YqTtwB7AWqKD95NGItwjJg%2FRachel_Rossin_portrait_2.jpg",
    vertical: "Market Analysis",
    author: "Daniel Kunitz",
    date: "Jan 4, 2017",
    subheader: "Nulla vitae elit libero, a pharetra augue. Vestibulum id ligula porta felis euismod semper.",
  },
  {
    layout: "split",
    title: "New York's Next Art District",
    url: "https://artsy-media-uploads.s3.amazonaws.com/ZR0wtJhg5Nez7Vm8uCP8Nw%2FDSC_0720-Edit-2.jpg",
    vertical: "Market Analysis",
    author: "Daniel Kunitz",
    date: "Jan 4, 2017",
    subheader: "Nulla vitae elit libero, a pharetra augue. Vestibulum id ligula porta felis euismod semper.",
  },
  {
    layout: "full",
    title: "New York's Next Art District",
    url: "https://artsy-media-uploads.s3.amazonaws.com/ZR0wtJhg5Nez7Vm8uCP8Nw%2FDSC_0720-Edit-2.jpg",
    vertical: "Market Analysis",
    author: "Daniel Kunitz",
    date: "Jan 4, 2017",
    subheader: "Nulla vitae elit libero, a pharetra augue. Vestibulum id ligula porta felis euismod semper.",
  },
]
