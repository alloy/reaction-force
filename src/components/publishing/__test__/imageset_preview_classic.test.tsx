import ImageSetPreviewClassic from "../imageset_preview_classic"
import { Images } from "./fixtures"

import * as React from "react"
import * as renderer from "react-test-renderer"

it("renders properly", () => {
  const imageset = renderer.create(<ImageSetPreviewClassic images={Images} />).toJSON()
  expect(imageset).toMatchSnapshot()
})
