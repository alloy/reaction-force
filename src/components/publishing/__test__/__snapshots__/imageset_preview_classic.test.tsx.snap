// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="imageset-preview file__Wrapper-s1l6im62-0 krPgOf"
>
  <div
    className="imageset-preview__container"
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <img
      className="imageset-preview__image"
      height={150}
      src="https://d32dm0rphc51dk.cloudfront.net/0aRUvnVgQKbQk5dj8xcCAg/larger.jpg"
      style={
        Object {
          "marginRight": 10,
        }
      }
    />
    <img
      className="imageset-preview__image"
      height={150}
      src="https://artsy-media-uploads.s3.amazonaws.com/co8j2xq40ROMyBrJQm_4eQ%2FDafenOilPaintingVillage_AK03.jpg"
      style={
        Object {
          "marginRight": 10,
        }
      }
    />
    <img
      className="imageset-preview__image"
      height={150}
      src="https://d32dm0rphc51dk.cloudfront.net/CpHY-DRr7KW0HGXLslCXHw/larger.jpg"
      style={
        Object {
          "marginRight": 10,
        }
      }
    />
  </div>
  <div
    className="imageset-preview__remaining file__Remaining-s1l6im62-2 kUzkwY"
  >
    <div
      className="imageset-preview__icon-container file__IconContainer-s1l6im62-3 gaazlU"
    >
      <svg
        data-name="imageset"
        id="imageset"
        viewBox="0 0 44 44"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M6760.67,551.72h-32v-32h32v32Zm-30-2h28v-28h-28v28Z"
          transform="translate(-6728.67 -507.72)"
        />
        <polygon
          points="38 37 36 37 36 8 7 8 7 6 38 6 38 37"
        />
        <polygon
          points="44 31 42 31 42 2 13 2 13 0 44 0 44 31"
        />
      </svg>
      <div
        className="imageset-preview__length"
        style={
          Object {
            "left": 8,
            "position": "absolute",
            "top": 14,
          }
        }
      >
        <div
          className="file__Text-s1l6im62-1 cWalav"
        >
          3
        </div>
      </div>
    </div>
    <div
      className="imageset-preview__text file__Text-s1l6im62-1 cWalav"
    >
      Enter Slideshow
    </div>
  </div>
</div>
`;
